examples/tests/valid/test_without_bring.w

```diff
@@ -0,0 +1,3 @@
+test "hello test" {
+  assert(true);
+}
```

Comment:
Adding new tests increases our CI build times by a bit, we could also just remove `bring cloud;` from [this file](https://github.com/winglang/wing/blob/main/examples/tests/valid/std_string.w)

Reply:
Adding new tests should be "free". If it's not, let's run do more parallelism.

Reply:
At any rate. I am leaving this test because I want to make sure we have explicit coverage for this case.